Example/StripeConnectExample/StripeConnect Example.xcodeproj/project.pbxproj

```diff
@@ -27,6 +27,8 @@
 		41810D712C89F4CD00F10EB7 /* AppearanceSettings.swift in Sources */ = {isa = PBXBuildFile; fileRef = 41810D702C89F4CD00F10EB7 /* AppearanceSettings.swift */; };
 		41810D762C8A005D00F10EB7 /* AppearanceInfo.swift in Sources */ = {isa = PBXBuildFile; fileRef = 41810D752C8A005D00F10EB7 /* AppearanceInfo.swift */; };
 		41810D782C8A006F00F10EB7 /* AppSettings+AppearanceInfo.swift in Sources */ = {isa = PBXBuildFile; fileRef = 41810D772C8A006F00F10EB7 /* AppSettings+AppearanceInfo.swift */; };
+		41810D7F2C8A0BAD00F10EB7 /* OFL.txt in Resources */ = {isa = PBXBuildFile; fileRef = 41810D7D2C8A0BAD00F10EB7 /* OFL.txt */; };
+		41810D802C8A0BAD00F10EB7 /* Matemasie-Regular.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 41810D7E2C8A0BAD00F10EB7 /* Matemasie-Regular.ttf */; };
```

Comment:
nit: The OFL and font files should be grouped into a folder matching the font name (e.g. `Resources/Fonts/Matemasie/`), just so it's extra clear that the license pertains to the specific font if someone stumbles upon it in our repo.